recipes-ni/ni-base-system-image-tests/files/fs_permissions_known.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+import csv
+import os
+import sys
+import glob
+import stat
+import pwd
+import grp
+import fs_permissions_shared
+
+def log_mismatch(col, exp, act, path, logger):
+    logger.log(f'ERROR: Expected {col} \'{exp}\' but got \'{act}\' for path {path}.')
+
+class Permissions:
+    def __init__(self, file_name):
+        if not os.path.exists(file_name):
+            self.path = None
+            self.mode = None
+            self.user = None
+            self.group = None
+            self.link = None
```

Comment:
There's no `else` here. Does the path need to exist for what follows? If not, why the conditional?

Reply:
I'm missing a return here. Later on I check for path to be None, so at least path needs to be None bc I log an issue later using that, but I suppose we don't need to set the others